lib/views/frontend/spree/checkout/payment/_paypal_braintree.html.erb

```diff
@@ -22,7 +22,9 @@
       line1: '<%= address.address1 %>',
       line2: '<%= address.address2 %>',
       city: '<%= address.city %>',
-      state: '<%= address.state.name %>',
+      <% if Spree::Config[:address_requires_state] %>
+        state: '<%= address.state.name %>',
+      <% end %>
```

Comment:
I think we should always transfer the state if we have one, on matter what this config is.